vueless.config.ts

```diff
@@ -6,9 +6,9 @@ export default {
   ringOffset: 0,
   ringOffsetColorLight: "#ffffff", // white
   ringOffsetColorDark: "#111827", // gray-900
-  "rounding-sm": 4,
+  roundingSm: 4,
```

Comment:
you need to change at all relates places, not only in here. It's not working now. Please test it before sending MR for review.

add `rounded-dynamic-lg` to the `UCard` and `rounded-dynamic-sm` to the `UCheckbox`.